examples/angular_bazel_architect/BUILD.bazel

```
@@ -102,7 +102,10 @@ architect_test(
 
 architect_test(
     name = "e2e",
-    args = ["frontend:e2e"],
+    args = [
+        "frontend:e2e",
+        "--no-webdriver-update",
```

The architect CLI, doesn't transform kebabed cases args to camel. But the schema validation needs camel cased properties.

```suggestion
        "--webdriverUpdate=false",
```

---

hmm still fails
```
Error: Schema validation failed with the following errors:
  Data path ".webdriverUpdate" should be boolean.
```

---

I looked at this and I pushed a commit (I hope you don't mind) which should address the problem.